src/Test/Monad/Cont.hs

```diff
@@ -12,36 +12,46 @@ import Test.QuickCheck.HigherOrder (Equation(..))
 -- | 'callCC' has no effects other than passing the continuation to the provided
 -- function.
 --
---  > callCC (const x) = x
+-- @
+--  'callCC' ('const' x) = x
+-- @
 callCC_const :: forall m a. MonadCont m => m a -> Equation (m a)
 callCC_const m = callCC (const m) :=: m
 
 -- | The continuation given returns the value passed to it, and not some other
--- one. 
+-- one.
 --
--- > callCC ($ x) = pure x 
+-- @
+-- 'callCC' ('$' x) = 'pure' x
+-- @
 callCC_id :: forall m a. MonadCont m => a -> Equation (m a)
 callCC_id a = callCC ($ a) :=: pure a
 
 -- | The continuation given returns the value passed to it, whether it's pure or
 -- in the \'context\' of some effect.
 --
--- > callCC ((>>=) m) = m
+-- @
+-- 'callCC' (('>>=') m) = m
+-- @
 callCC_bind :: forall m a. MonadCont m => m a -> Equation (m a)
 callCC_bind m = callCC ((>>=) m) :=: m
 
 -- | The return type of the continuation is effectively 'm Void'; it will never
 -- actually \'return\'.
 --
--- > callCC f = callCC (f . (fmap . fmap) absurd)
+-- @
+-- 'callCC' f = 'callCC' (f . ('fmap' . 'fmap') 'absurd')
```

Comment:
```suggestion
-- 'callCC' f = 'callCC' (\\k -> f ('absurd' '.' k))
-- @
```

Now that I think about it, the pointful version actually seems nicer. You can also change the actual code below (make sure it typechecks!)